sql-plugin/src/main/scala/org/apache/spark/sql/rapids/ExternalSource.scala

```diff
@@ -121,6 +130,10 @@ object ExternalSource {
   }
 
   def getScans: Map[Class[_ <: Scan], ScanRule[_ <: Scan]] = {
+    if (showErrorMessage) {
+      logWarning("WARNING: Avro library not found by the RAPIDS plugin. If needed, please use " +
+          "--jars to add it")
```

Comment:
we need to document that --jars or --packages is the preferred way of deploying the plugin jar, specifically for Avro.

Reply:
Let us drop WARNING because it's already part of the logging category. The message should talk about the plugin jars, since this is the actual problem, not Avro:

"Avro library not found by the RAPIDS plugin. The Plugin jars are likely deployed using a static classpath spark.driver/executor.extraClassPath. Consider using --jars or --packages instead."